### PR TITLE
Examples: SDKs

### PR DESCRIPTION
**Pull request by samijaber**
## Description

Changes:
- delete tons of irrelevant code in svelte/vue examples
- improve READMEs
- add Vue 3 example
- rename wrapper components to `DynamicallyRenderBuilderPage`

Ended up finding bugs and fixing them:
- Vue overrides were not being respected

### CODE REVIEW

**Comment by samijaber** [comment]
@steve8708 might be hard to see in this PR, but I factored in your feedback on cleaning up examples. 
- Deleted a ton of unused files/JSX/CSS in all the examples,
- renamed the 1 abstraction to `DynamicallyRenderBuilderContent` across all examples, 
- simplified logic as much as possible
- improved READMEs to point to relevant code

**Review by steve8708 — APPROVED** [review]

File: examples/vue/vue-2/src/components/DynamicallyRenderBuilderPage.vue [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      apiKey: BUILDER_PUBLIC_API_KEY,
+      options: getBuilderSearchParams(
+        convertSearchParamsToQueryObject(new URLSearchParams(window.location.search))
+      ),
+      userAttributes: {
+        urlPath: window.location.pathname,
+      },
+    }).then(res => {
+      this.content = res;
+      this.canShowContent = this.content || isEditing() || isPreviewing();
```
Comment (steve8708) [inline examples/vue/vue-2/src/components/DynamicallyRenderBuilderPage.vue:79]: thought we had a convo about we were going to purely use `isPreviewing()`? that's the intent of the new SDKs, really need things to be as simple as possible

Comment (steve8708) [reply examples/vue/vue-2/src/components/DynamicallyRenderBuilderPage.vue:79]: (aka no more need for isEditing() - the editor is also a preview)

File: examples/vue/vue-3/src/components/DynamicallyRenderBuilderPage.vue [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      return REGISTERED_COMPONENTS;
+    },
+  },
+  mounted() {
+    getContent({
+      model: 'page',
+      apiKey: BUILDER_PUBLIC_API_KEY,
+      options: getBuilderSearchParams(
+        convertSearchParamsToQueryObject(new URLSearchParams(window.location.search))
+      ),
```
[the remark this answers is not in the document]
Comment (samijaber) [reply examples/vue/vue-3/src/components/DynamicallyRenderBuilderPage.vue:70]: I fully agree with you, it's been on my list for a while, just haven't gotten to do it yet: https://builder-io.atlassian.net/browse/ENG-2476

- It's a question of improving `getBuilderSearchParams`, but we _do_ need a way for the user to provide these params for SSR, or else the SDK won't work for SSR. 
- agreed that the `canShowContent` logic should be fully baked into `RenderContent`, and we should stop exporting `isPreviewing` and `isEditing`altogether

We can discuss solutions further in JIRA/Slack, but I've always been on board to remove this logic. 😄

File: examples/vue/vue-2/src/components/DynamicallyRenderBuilderPage.vue (hunk 1 above)
Comment (samijaber) [reply examples/vue/vue-2/src/components/DynamicallyRenderBuilderPage.vue:79]: They aren't quite the same thing: 
image

But I agree with you on simplifying this as well. Just requires some more changes. We can also bake this logic into `RenderContent` and remove the need to export these 2 helpers altogether

File: examples/vue/vue-3/src/components/DynamicallyRenderBuilderPage.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (steve8708) [reply examples/vue/vue-3/src/components/DynamicallyRenderBuilderPage.vue:70]: perfect, added a couple notes in that ticket 👍

**Pull request merged**